pkg/engine/funcs_test.go

```diff
@@ -56,7 +56,7 @@ func TestFuncs(t *testing.T) {
 	}, {
 		// Regression for https://github.com/helm/helm/issues/2271
 		tpl:    `{{ toToml . }}`,
-		expect: "[mast]\n  sail = \"white\"\n",
+		expect: "\n[mast]\n  sail = \"white\"\n",
```

Comment:
Why is the expected response from `toToml` changing? If someone is today using `toToml` then they may have issues in their template if they moved over to this due to a new line being added at the start.

Reply:
I agree that if you expect exactly the same output, things may break. Though, as long as the TOML that is exported is valid, any parser should ignore new lines, so don't see this as a big issue.